### PR TITLE
Add a script that builds a minimal developer environment

### PR DESCRIPTION
**Pull request by ciprianc**
The script will:
 - pull down a postgresql + postgis container and run it
 - build a container for the django code, based on the Dockerfile in the repo (so it resembles prod)
 - link the containers
 - runs a django migrate
 - mount the work dir inside the container so a developer can use its favourite IDEs to change code that's running inside the container (django picks up on the changes and restarts itself automatically)

Documentation coming up shortly in a different PR

### CODE REVIEW

**Comment by ashb** [comment]
Could you put the PR description in a comment in the script? It's a useful starting point on docs

**Comment by ciprianc** [comment]
Yup, sounds like a good idea, a sec

**Comment by ciprianc** [comment]
Closing and will reopen once I update the readme too.

File: README.md [hunk 1]
```diff
@@ -6,55 +6,66 @@ Address Finder
...
 ```
+Note: If you want to execute custom commands inside the docker container you can easily do so by opening up a new shell and running the commands in the container. There's an example on how to do so in the next section (Creating a superuser)
 
-### Create database schema and superuser
+### Create a superuser (optional)
+If you need to use the django built-in admin interface you would need to create an user. We can do so by executing the createsuperuser command in the running container:
 
 ```bash
-$ ./manage.py syncdb
+$ docker exec -ti  --rm postcode-web ./manage.py createsuperuser
```
Comment (aldavidson) [inline README.md]: this doesn't work for me:
```
flag provided but not defined: --rm
```

File: README.md [hunk 2]
```diff
@@ -6,55 +6,66 @@ Address Finder
...
 ```
 
 ### Download and Import Data
+In order to download and import the data we must execute the `download_and_import_all` command. We can either do that as in the "Create a superuser" section, or get a shell inside the container like so:
 
 ```bash
+$ docker exec -ti --rm postcode-web /bin/bash
```
Comment (aldavidson) [inline README.md]: this doesn't work for me:
```
flag provided but not defined: --rm
```

File: README.md [hunk 3]
```diff
@@ -6,55 +6,68 @@ Address Finder
...
+$ docker exec -ti postcode-web ./manage.py createsuperuser
 ```
 
 ### Download and Import Data
+In order to download and import the data we must execute the `download_and_import_all` command. We can either do that as in the "Create a superuser" section, or get a shell inside the container like so:
 
 ```bash
+$ $(boot2docker shellinit)
+$ docker exec -ti postcode-web /bin/bash
 $ ./manage.py download_and_import_all
```
Comment (aldavidson) [inline README.md]: change to:

$ ./scripts/download_and_import_all.sh

**Pull request merged**